lib/identity/stateTransitions/identityCreateTransition/IdentityCreateTransition.js

```diff
@@ -1,27 +1,26 @@
-const hash = require('../../../util/hash');
 const AbstractStateTransition = require('../../../stateTransition/AbstractStateTransition');
 const stateTransitionTypes = require('../../../stateTransition/stateTransitionTypes');
 const IdentityPublicKey = require('../../IdentityPublicKey');
-const Identifier = require('../../../identifier/Identifier');
+const AssetLock = require('../assetLock/AssetLock');
 
 class IdentityCreateTransition extends AbstractStateTransition {
   /**
-   * @param {RawIdentityCreateTransition} [rawIdentityCreateTransition]
+   * @param {RawIdentityCreateTransition} rawStateTransition
    */
-  constructor(rawIdentityCreateTransition = {}) {
-    super(rawIdentityCreateTransition);
+  constructor(rawStateTransition) {
+    super(rawStateTransition);
 
     this.publicKeys = [];
 
-    if (Object.prototype.hasOwnProperty.call(rawIdentityCreateTransition, 'publicKeys')) {
+    if (Object.prototype.hasOwnProperty.call(rawStateTransition, 'publicKeys')) {
       this.setPublicKeys(
-        rawIdentityCreateTransition.publicKeys
+        rawStateTransition.publicKeys
           .map((rawPublicKey) => new IdentityPublicKey(rawPublicKey)),
       );
     }
 
-    if (Object.prototype.hasOwnProperty.call(rawIdentityCreateTransition, 'lockedOutPoint')) {
-      this.setLockedOutPoint(rawIdentityCreateTransition.lockedOutPoint);
+    if (Object.prototype.hasOwnProperty.call(rawStateTransition, 'assetLock')) {
```

Comment:
Why did we decide to rename it back again?

Reply:
Now, `assetLock` is an object with transaction, oupointIndex, and proof (which is an object too, InstantAssetLockProof) represented by AssetLock class. So `lockedOutPoint` is not representative anymore.